test/legacy/aix_services.yml

```diff
@@ -0,0 +1,24 @@
+---
+- name: Services/Subsystems tests for AIX
+  hosts: localhost
+  connection: local
+  tasks:
+    - name: spooler shutdown
+      service:
+        name: spooler
+        state: started
+
+    - name: stopping sendmail
+      service:
+        name: sendmail
+        state: stopped
+
+    - name: starting sendmail
+      service:
+        name: sendmail
+        state: started
+
+    - name: starting an inexistent subsystem and group subsystem
+      service:
+        name: fakeservice
```

Comment:
wouldn't this fail? do we want it to fail or should that be handled gracefully by an assertion?

Reply:
It looks like this should fail for a nonexistent service as decided in #35293 the correct option here is to use either ignore_errors: true or a failed_when statement to prevent an error.